src/m_menu.c

```diff
@@ -4240,6 +4245,9 @@ setup_menu_t gen_settings6[] = {
   {"Screen flashes", S_ONOFF|S_STRICT, m_null, M_X, M_SPC,
    {"palette_changes"}},
 
+  {"Invulnerability Mode", S_CHOICE|S_STRICT, m_null, M_X, M_SPC,
+   {"invul_mode"}, 0, R_InvulMode, str_invul_mode},
```

Comment:
Not sure where to put this or how to name it.

Reply:
I'm not sure. This option is purely cosmetic and doesn't affect playsim compatibility.

Reply:
My other comment was lost, so let's try again. Suggestions:
"Invulnerability Effect"
"Vanilla", "MBF", "MBF Gray"
- Vanilla: No sky fix.
- MBF: Uses sky fix.
- MBF Gray: Uses sky fix, everything is gray.

Reply:
> MBF: Uses sky fix.

I don't like the name "MBF", not many users know that sky fix was invented in MBF.

> MBF Gray: Uses sky fix, everything is gray.

I think vanilla invul is better default.

Reply:
I think we can say that if people prefer the vanilla sky, then they also prefer the vanilla color map?

Reply:
> I think we can say that if people prefer the vanilla sky, then they also prefer the vanilla color map?

Ok, I will enable "sky fix" for "gray" colormap. How should we name these options, what do you think?

Reply:
Oh, I'm bad at this. But I would go with @ceski-1's suggestion as well.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1542/commits/202cf94db89c7b5423fdebac54f7ccd5f9360e95